Code/Tools/Libs/GuiFoundation/PropertyGrid/Implementation/PropertyBaseWidget.cpp

```diff
@@ -1253,18 +1244,34 @@ void ezQtPropertyTypeContainerWidget::UpdateElement(ezUInt32 index)
     }
 
     const ezRTTI* pCommonType = ezQtPropertyWidget::GetCommonBaseType(ResolvedObjects);
+
+    // Label
     {
-      // Label
       ezStringBuilder sTitle;
       sTitle.Format("[{0}] - {1}", m_Keys[index].ConvertTo<ezString>(), ezTranslate(pCommonType->GetTypeName()));
       elem.m_pSubGroup->SetTitle(sTitle);
     }
+
+    // Icon
     {
-      // Icon
       ezStringBuilder sIconName;
       sIconName.Set(":/TypeIcons/", pCommonType->GetTypeName());
       elem.m_pSubGroup->SetIcon(ezQtUiServices::GetCachedIconResource(sIconName.GetData()));
     }
+
+    // help URL
+    {
+      QString url = ezTranslateHelpURL(pCommonType->GetTypeName());
+
+      if (!url.isEmpty())
+      {
+        ezQtElementGroupButton* pHelp = new ezQtElementGroupButton(elem.m_pSubGroup->GetHeader(), ezQtElementGroupButton::ElementAction::Help, this);
```

Comment:
Don't create any widgets in this function. This function will be called multiple times for the same instance and other instances  so you accumulate buttons constantly and trigger layout / repaint unnecessarily. Better follow the example of the delete button and create the button at the start and then just hide it depending on whether it is needed for the current selection.

Reply:
Is this better? Working with this part of the code is a bit tricky, I'm not very familiar with the structure.